packages/daemon/src/graphql/objects.js

```diff
@@ -275,6 +284,9 @@ export const app = new GraphQLObjectType({
         return ctx.openVault.apps.getUpdate(self.id)
       },
     },
+    installationState: {
+      type: installationState,
```

Comment:
I think this state is always present in the vault so the type here should be `new GraphQLNonNull(installationState)`